### ISSUE TITLE
Compact pagination on mobile phone is not usable

### ISSUE THREAD

**Issue by rvsia** [patternfly/patternfly-react#3665]
**Describe the issue. What is the expected and unexpected behavior?**

When using compact pagination on smaller screen, it's not usable.

![compactpag](https://user-images.githubusercontent.com/32869456/73941134-f948c900-48ec-11ea-9261-cf0ba16e4e47.gif)

Expected:

An ability to switch pages.

**Please provide the steps to reproduce. Feel free to link CodeSandbox or another tool.**

1. Set compact navigation
2. Decrease size of your browser's window
3. Try to change the page

**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**

A bug. It's not blocking us.

**What is your product and what release version are you targeting?**

Sources UI ASAP.
It affects all other Insights applications that are using the compact pagination.

cc @Hyperkid123 @karelhala

**Comment by karelhala** [comment]
I think that this should be promoted all the way to styles as well. We could update the pagination with some new usecases. If you want we can give them to you so you will know how users are using it and what could be improved.

**Comment by tlabaj** [comment]
cc @mcarrano @mcoker

**Comment by mcarrano** [comment]
@karelhala @rvsia So the way this is supposed to work is illustrated here: https://www.patternfly.org/v4/design-guidelines/usage-and-behavior/pagination. It was always the design intent that there would always be a pagination control both above and below the list or table. That's why when you look at the Pagination component you will see Top and Bottom variants that respond differently.

**Comment by rvsia** [comment]
@mcarrano I understand that, but I think It will be a very confusing and inconsistent user experience: in a full view it works, on a mobile not. And with so many smartphones out there, we should carefully take care of these users too.

I can imagine that when user clicks on this component in the compact mode, some button could appear on the screen. Or it could work as a dropdown. There are so many ways how to preserve the compact size and the functionality at the same time and I don’t see any reason why to disable it.

**Comment by mcarrano** [comment]
Fair point @rvsia . In any case, this is a design and not a React issue, so I will transfer to patternfly-design for further consideration.
